### ISSUE TITLE
[sc-select] - Number are parsed as string

### ISSUE THREAD

**Issue by etiennedemoulin** [ircam-ismm/simple-components#11]
```js
<sc-select
  .options=${[0, 1, 2, 3.4]}
  @change=${e => console.log(typeof e.target.value)}
></sc-select>
```
return string - should return number

**Comment by b-ma** [comment]
nice catch, I think I see where the problem comes from

(for information, I created a new repo there: https://github.com/ircam-ismm/sc-components/, I will archive this one)

**Comment by b-ma** [comment]
fixed in 3.0.0-alpha.8
